src/storage-client/src/types/sources.rs

```diff
@@ -1421,7 +1428,7 @@ impl SourceDesc {
 }
 
 #[derive(Arbitrary, Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
-pub enum SourceConnection {
+pub enum GenericSourceConnection {
```

Comment:
not sure i understand the name change to `GenericSourceConnection`, i think leaving it as is was fine, but i suppose we can iterate on this

Reply:
There is a trait with the same name that is now in the same file and it was annoying disambiguating them so I renamed this